cfpb/capital-framework
# Update link hover color to Dark Pacific for accessible contrast

Our current link hover color (Pacific 60) does not meet WCAG2 Level AA contrast requirements. This PR fixes that by changing it to Dark Pacific.

/cc @designlanguage @nataliafitzgerald 

## Changes

- Link hover color changed to Dark Pacific


## Testing

Follow [the steps to testing with the docs](https://github.com/cfpb/capital-framework/blob/master/CONTRIBUTING.md#testing-in-the-documentation-site). Presuming you have Bundler set up already:

1. Pull this branch
1. `gulp`
1. `cd docs`
1. `bundle exec jekyll serve`
1. Visit http://localhost:4000/components/cf-core/#default-links and see the new darker link hover color (cf. https://cfpb.github.io/capital-framework/components/cf-core/#default-links)


## Checklist

- [x] PR has been tagged with the type of change it includes
      (breaking, enhancement, bug, documentation or internal)
- [x] Project documentation has been updated in this PR with the `gulp docs` command
